configs/components/fesom/fesom-2.1.yaml

```diff
@@ -11,32 +11,27 @@ clean_command: ${defaults.clean_command}
 
 use_icebergs: False
 
+with_wiso: false
+wiso_code: false
+
 choose_version:
   2.1-unstable:
     branch: master
     namelist_dir: "${model_dir}/config/"
   '2.1':
     branch: "2.1.0"
+  2.1-awiesm-2.1:
```

Comment:
I still hate this name, but I do not see a more elegant solution: `<model_version>-<setup_name>-<setup_version>` Maybe we put that as a comment somewhere so people reading it know why we chose it that way?

Reply:
we can just name it `awiesm-2.1` since it is already in the `fesom-2.1.yaml`.

Reply:
Well, the filename for the configuration default is a first step, but I would rather have a unique ID directly on the source code, so: it would be best to here be consistent. It's a matter of 5 minutes, but I would rather get it right once and then never need to touch it again. Proposal would be:

* Atmosphere: `ECHAM-6.3.05p2` on branch `6.3.05p2-awiesm-2.1` (that's the merged version with all the fancy things inside: wiso+ice sheets+...)
* Ocean: `FESOM-2.1` (with added wiso and other "bonus" features) on branch `2.1-awiesm-2.1` (pico+wiso+icebergs+...)
* Coupler: whatever it currently is (no one had hiccups in the coupler yet), but, we give a new tag with `<oasis_number>-awiesm-2.1`

I know it's annoying, and I know it is also the last thing any one of us wants to spend more than 5 minutes on, but, it will make communication about maintainability much easier in the long term.

For those not working directly on code: please just a thumbs up/thumbs down, and the esm-tools guys will take care of it.

@mwerner-awi @xshi-awi @ackerlar

Reply:
Thumps up from my side. I only see a bit of redundance, when we will have versions `2.1-awiesm-2.1`, `2.1-awiesm-2.2`, `2.1-awiesm-3.0`, ... which comprise basically the same fesom version. Or do I get this wrong?

Reply:
There is no `or` or `and` in the yamls, but there are alternative ways of simulating that behavior if necessary.

Reply:
Ok, I could write more details, but: please don't bend the YANL further.